### PR TITLE
Add `bolt` (settings variant)/`drill` icons

### PR DESCRIPTION
**Pull request by danielbayley**
Related to #1369.

Edit: Leaving `screw` icons here, for reference…

preview

### CODE REVIEW

**Comment by ericfennis** [comment]
Can you add use cases to your PR?

**Comment by jguddas** [comment]
They don't feel 100% great.

**Comment by danielbayley** [comment]
> Can you add use cases to your PR?

@ericfennis `bolt` is obviously a commonly found alternative to `cog`/`gear` as `settings`… Other 2 felt like easy corresponding additions to https://lucide.dev/icons/categories#tools, like [hammer](https://lucide.dev/icons/hammer)/[wrench](https://lucide.dev/icons/wrench) etc, along with `drill`, which I hadn’t yet `push`ed… Also added metadata.

**Comment by ericfennis** [comment]
@danielbayley Thanks for you reply.
`Bolt` and `drill` has good use-cases, I doubt the `screw` and `screw-head`.
And can we try to make a simpler design for the `drill` icons. I looks a bit too dense to me.

File: icons/drill.svg [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M14 9c0 .6-.4 1-1 1H4a2 2 0 0 1-2-2V4a2 2 0 0 1 2-2h9c.6 0 1 .4 1 1Z" />
+  <path d="M18 6h4" />
+  <rect width="4" height="4" x="14" y="4" rx="1" />
```
Comment (karsa-mistmere) [inline icons/drill.svg:14]: Using a `<rect>` here has some scalability issues:
image

```suggestion
  <path d="M14 4h3a1 1 0 0 1 1 1v2a1 1 0 0 1-1 1h-3" />
```

**Pull request merged**